canonical-web-and-design/microk8s.io
# Build tutorials page

## Done

- Add tutorials page to nav
- Build tutorials page at `/tutorials`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/tutorials
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against [copy doc](https://docs.google.com/document/d/1Lpo7Od9VA1aHYmr9UVp02vCGOP5Zxmzx4zZE9YJwlQg/edit#)
- Check links 

## Issue / Card

Fixes [#3260](https://github.com/canonical-web-and-design/web-squad/issues/3260)
